.github/workflows/backwards-compatibility-check.yaml

```diff
@@ -0,0 +1,22 @@
+name: Backwards Compatibility Check
+on: [pull_request]
+jobs:
+  # More info at https://github.com/Roave/BackwardCompatibilityCheck.
+  backwards-compatibility-check:
+    runs-on: ubuntu-latest
+    # Disable this check by applying the 'breaking change allowed' label to the PR.
+    if: "!contains(github.event.pull_request.labels.*.name, 'breaking change allowed')"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
```

Comment:
Even though you're running this on PHP 7.4, I believe it's actually parsing using PHP 8.1 (since it's failing on a `READONLY` keyword syntax error, which was added in PHP 8.1).

I would suggest running it on PHP 8.0 and configuring it to use PHP 8.0 (since that's our highest supported version at the moment)

Reply:
I switched back to using the Docker image instead as it is the simplest. Regardless of what version we use to run the job, as you said because it parses using 8.1, the job will fail until the `ReadOnly` protobuf fix lands.